content/release-notes/2022-10-07.md

```diff
@@ -0,0 +1,13 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
```

Comment:
```suggestion
- Pageserver: Increased the default compaction_period setting to 20 seconds, affecting the per-tenant compaction background process. With numerous projects, the previous default setting of 1 second caused excessive CPU usage.
```

Reply:
besides the increased CPU usage on pageserver that is not necessarily affecting the users, were there symptoms that users could feel that may be relieved with this?

Reply:
Looks like the issue was found in captest. 
https://github.com/neondatabase/neon/issues/2542
https://github.com/neondatabase/neon/pull/2527
I have revised the entry with more details.